PyRDF/RDataFrame.py

```diff
@@ -78,6 +78,31 @@ def _get_vector_from_list(self, arg):
 
         return reqd_vec
 
+    def get_num_entries(self):
+        """
+        Gets the number of entries in
+        the given dataset.
+
+        """
+        if isinstance(self.args[0], int):
+            # If there's only one argument
+            # which is an integer, return it.
+            return self.args[0]
+
```

Comment:
Can we add here:
```python
elif isinstance(self.args[0]), ROOT.TTree):
    return self.args[0].GetEntries()
```

Reply:
Since they are repeated several times, can we have local variables for:
```python
first_arg = self.args[0]
sec_arg = self.args[1]
```
**right before** the first time they are used?

Reply:
Sounds good @etejedor . I'll make both these changes 👍

Reply:
Shall I also add a test for the `ROOT.TTree` case @etejedor ?

Reply:
Yes, please add another test.

Reply:
👍